src/js/components/legend/Legend.tsx

```diff
@@ -50,6 +59,24 @@ const Legend = (): JSX.Element => {
   const [visibleLayersToShow, setVisibleLayersToShow] = useState<LayerProps[]>(
     []
   );
+
+  useEffect(() => {
+    const handleResize = () => {
+      setWindowDimensions(getWindowDimensions());
+    };
+
+    window.addEventListener('resize', handleResize);
+    window.removeEventListener('resize', handleResize);
+  }, []);
+
+  useEffect(() => {
+    const { width } = windowDimensions;
+
+    if (width < 475 && hideLegend) {
+      setLegendOpen(hideLegend);
```

Comment:
Can you add a small comment here of why we are doing this hide Legend checks for the future? I feel like people who will be working on this code at some point may get confused. Couple lines would suffice.